src/App.tsx

```diff
@@ -1,61 +1,132 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import React, { useState } from 'react';
+import { User } from './types/User';
+import { Todos } from './types/Todos';
 
 export const App = () => {
+  const getUserById = (userId: number): User | null =>
+    usersFromServer.find(user => user.id === userId) || null;
+
+  const initialTodos: Todos[] = todosFromServer.map(todo => ({
+    ...todo,
+    user: getUserById(todo.userId),
```

Comment:
The `getUserById` function can return `null` if a user is not found. Ensure that the `user` property in the `initialTodos` array handles this case to avoid potential runtime errors.